README: distinguish announcements section (#7568)
actions/runner-images: README.md

There are no migrations in progress.

> ## Announcements
> 
> You can also view notable upcoming changes by viewing the [Announcement](https://github.com/actions/runner-images/labels/Announcement) label.
> 

Reviewer: Wondering if we should rename `Ongoing migrations` to `Announcements` and when we have a migration we can simply make an announcement issue for it. wdyt?

```suggestion
## Announcements

See notable upcoming changes by viewing the issues with the [Announcement](https://github.com/actions/runner-images/labels/Announcement) label.

```

Author: @samvantran works for me
